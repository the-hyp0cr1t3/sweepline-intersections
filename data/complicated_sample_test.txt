10
10 5 2 9
2 7 4 9
5 9 4 2
8 8 8 1
9 8 9 5
6 5 2 5
3 2 6 6
2 4 3 2
9 1 6 4
7 1 12 3
