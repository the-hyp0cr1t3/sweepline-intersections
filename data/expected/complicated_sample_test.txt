11
3.0000000000 2.0000000000 7 8
3.3333333333 8.3333333333 1 2
4.2352941176 3.6470588235 3 7
4.4285714286 5.0000000000 3 6
4.8000000000 7.6000000000 1 3
5.2500000000 5.0000000000 6 7
8.0000000000 1.4000000000 4 10
8.0000000000 2.0000000000 4 9
8.0000000000 6.0000000000 1 4
8.4285714286 1.5714285714 9 10
9.0000000000 5.5000000000 1 5

